web/src/engine/websites/RawOtaku.ts

```diff
@@ -0,0 +1,40 @@
+import { Tags } from '../Tags';
+import icon from './RawOtaku.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as Liliana from './decorators/Liliana';
+
+export const pageScript = `
+    new Promise(resolve => {
+        const elementid = '#' + manga.lang + '-chapters li[data-number="'+ manga.name +'"]';
+        const chapterid = document.querySelector(elementid).dataset.id;
+        $.get(
```

Comment:
Isn't the error handling function missing in this `JQuery` [get](https://api.jquery.com/jQuery.get/) request?
May also consider to use `.done()` and `.fail()` fluent methods which are equivalent to `Promise.then` and `Promise.catch`.

Reply:
I merely copied the website code. fine then.